pkg/config/config.go

```diff
@@ -93,6 +93,8 @@ type OverlayOptionsConfig struct {
 	MountProgram string `toml:"mount_program"`
 	// Size
 	Size string `toml:"size"`
+	// Inodes is used to set a maximum inodes of the container image.
+	Inodes string `toml:"inodes"`
```

Comment:
Do we wanna indicate that it's a "maximum" by renaming to `MaxInodes` or `InodesMax`? OTOH right now it matches better to the overlay option `inodes=`

Reply:
@rhatdan WDYT about rename it with max mentioned?

Reply:
I don't think people will be onfused, and we don't change size to MaxSize.